frameworks/elastic/src/main/dist/svc.yml

```diff
@@ -6,7 +6,8 @@ pods:
     count: 3
     {{#ENABLE_VIRTUAL_NETWORK}}
     networks:
-      dcos:
+      {{VIRTUAL_NETWORK_NAME_MASTER}}:
```

Comment:
Why are these distinct network names? Will Elastic work if it has totally different networks for different node types? (I'd assume not?)